cmake/Modules/CompilerOptions.cmake

```diff
@@ -1,13 +1,12 @@
 include_guard(GLOBAL)
 
-include(DetectCompiler)
```

Comment:
I still have to make the difference between GNU and CLANG because libsigmf is GNU specific. How does cmake resolves this?